test/framework/src/main/java/org/elasticsearch/action/support/replication/ClusterStateCreationUtils.java

```diff
@@ -342,7 +342,8 @@ public static ClusterState buildServerlessRoleNodes(
         String indexName,
```

Comment:
Update the comment up there ^

Reply:
Done in 6dbe7720786d30e8777640a94535a7962635c139